pydis_site/apps/content/README.md

```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
+relatively little code in there, most of it is concerned with serving our
+content.
+
+The markdown files hosting our content can be found in the
+[`resources/`](./resources) directory, and contain
+
+
+## Contributing pages
+
+Contributing pages is covered extensively in our online guide, which you can
+find
+[here](https://www.pythondiscord.com/pages/guides/pydis-guides/how-to-contribute-a-page/)
+or alternatively read it directly at
+[`resources/guides/pydis-guides/how-to-contribute-a-page.md`](./resources/guides/pydis-guides/how-to-contribute-a-page.md).
+
+
+## Directory structure
+
+Let's look at the structure in here:
+
+- `resources` contains the static Markdown files that make up our site's
+  [pages](https://www.pythondiscord.com/pages/)
+
+- `migrations` contains standard Django migrations. As the `content` app
+  contains purely static markdown files, no migrations are present here.
+
+- `tests` contains unit tests for verifying that the app works properly.
+
+- `views` contains our Django views generating and serving the pages from the
+  input markdown.
+
+As for the modules, apart from the standard Django modules in here, the
+`utils.py` module contains utility functions for discovering markdown files to
+serve on the filesystem.
```

Comment:
I don't think this clarification is necessary, and it introduces ambiguity: are files being served on the filesystem, or are files, which are on the filesystem, being served? Of course the former doesn't even make sense, but it's confusing regardless.

```suggestion
serve.
```

Reply:
ca2c583b15d46d7821a03ec047678429f96c7e03